build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
     id 'org.springframework.boot' version '2.1.9.RELEASE'
     id 'io.spring.dependency-management' version '1.0.8.RELEASE'
     id 'com.github.ben-manes.versions' version '0.26.0'
+    id "org.flywaydb.flyway" version "6.1.0"
```

Comment:
Not sure if you need this plugin

Reply:
It's a flyway gradle plugin I found with useful tasks that you can run, such as `gradle flywayInfo` that prints the details and status information about all the migrations. 
I know you can do this from command line so don't necessarily need.